recipes/museotoolbox/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "museotoolbox" %}
+{% set version = "0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 30201a51d6033c1b191bd822741f7cbf073053926928d818ea874bfcad3f9917
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - gdal >=2
+    - joblib
+    - matplotlib
```

Comment:
this should be matplotlib-base

Reply:
I changed to matplotlib-base and all checks are successful :+1:  : https://github.com/conda-forge/staged-recipes/pull/10863/commits/fa24744da2ffd38c4e22d515644bd5604a4bb215
Thank you for your patience.